Makefile

```
@@ -0,0 +1,10 @@
+.PHONY: ifs docker clean
+
+ifs: *.go
+	go build -o docker ./...
+
+docker:
+	docker build -t istio/ifs docker
```

If we take the suggestion to move dockerfile to toplevel

```suggestion
	docker build -t istio/ifs .
```